data/_ui/tooltips.txt

```diff
@@ -1531,3 +1531,15 @@ tip "Show parenthesis"
 
 tip "Notify on destination"
 	`If you enter a system with a planet you must land on for a mission, plays a notification sound.`
+
+tip `Override "universal ramscoop"`
```

Comment:
This makes the gamerules visible to casual players. Is a casual player supposed to know what gamerules are?

I wouldn't expect the text "Override" to be visible, but I would expect this to be presented as a setting.

Reply:
I'm not suggesting a rename; I'm just suggesting to show nothing about gamerules in the UI. Content writers need to know what gamerules are, but I feel that casual players just don't need to know about them.

The main audience for the game is casual players, and I feel that exposing gamerules like this will make the game more oriented at "power players" and "contributors".